graphql/resolvers.go

```diff
@@ -4084,6 +4089,14 @@ func (r *ticketFieldsResolver) ResolutionName(ctx context.Context, obj *thirdpar
 
 func (r *Resolver) TicketFields() TicketFieldsResolver { return &ticketFieldsResolver{r} }
 
+func (r *taskResolver) Ami(ctx context.Context, obj *restModel.APITask) (*string, error) {
```

Comment:
Nit: I kind of feel like I wrote this when I was young and foolish but now it burns my eyes? AMI definitely should be capitalized 😓  if that's an easy refactor to do now

Reply:
It isn't the resolver definition is automatically defined by gqlgen. I didn't see any way to change it in the docs.

Reply:
Sorry what I meant was, the field _in_ APITask is Ami instead of AMI. if you change it there, it should also fix the resolver.

Reply:
I updated the field in APITask but the resolver field is actually a separate value that is generated by the [`schema.graphql`](https://github.com/evergreen-ci/evergreen/blob/d57ddbc00888b3e21356e65a8c9fde7d54075dbb/graphql/schema.graphql#L1044) file and `gqlgen` even though it maps to the APITask its not the same variable. I renamed it in APITask however.